com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderQueue.cs

```diff
@@ -110,6 +110,8 @@ internal static RenderQueueType MigrateRenderQueueToHDRP10(RenderQueueType rende
         public const int sortingPriortyRange = 50;
         public static int ClampsTransparentRangePriority(int value) => Math.Max(-sortingPriortyRange, Math.Min(value, sortingPriortyRange));
 
+        public const int meshDecalPriortyRange = 50;
```

Comment:
there is a typo: meshDecalPriortyRange +> meshDecalPriorityRange  typo was already in sortingPriortyRange :)

Reply:
note that here add a comment to say that it is a RendererPriority unlike sortingPriortyRange  which is a RenderQueue priority